src/test/java/com/hotel/pad/mappers/response/HotelAvailabilityResponseMapperTest.java

```diff
@@ -0,0 +1,70 @@
+package com.hotel.pad.mappers.response;
+
+import com.hotel.service.availability.HotelAvailabilityResponse;
+import com.hotel.service.common.AvailableHotelItem;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.opentravel.ota._2003._05.ErrorType;
+import org.opentravel.ota._2003._05.ErrorsType;
+import org.opentravel.ota._2003._05.OTAHotelAvailRS;
+import org.opentravel.ota._2003._05.SuccessType;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.atLeast;
+
+@RunWith(MockitoJUnitRunner.class)
+public class HotelAvailabilityResponseMapperTest {
+    @Mock
+    private HotelItemMapper hotelItemMapper;
+    @InjectMocks
+    private HotelAvailabilityResponseMapper hotelAvailabilityResponseMapper = new HotelAvailabilityResponseMapper();
```

Comment:
I think it will be enough if we just create reference for HotelAvailabilityResponseMapper(@InjectMocks). no need to create object.
Just check once